### PR TITLE
Widget2

### PR DESCRIPTION
**Pull request by wesj**

### CODE REVIEW

**Comment by wesj** [comment]
I think some things like strings and colors that I'm stealing from Fennec shouldn't be in here. Not sure how we handle that.

**Comment by EricEdens** [comment]
> I think some things like strings and colors that I'm stealing from Fennec
> shouldn't be in here. Not sure how we handle that.
> 
> We've copied a few things from MC so that the project can compile -- for
> example BrowserContract. During the export process (_grunt export --tree .._)
> we ignore those copied resources.

In the case of strings and colors, I expect we'd create these resources in
their own file (eg: gecko_strings), and then ignore that when doing the
export.

I can help with updating our grunt config in order to ignore those
resources.

**Comment by wesj** [comment]
I updated this to get rid of the branding dir entirely. I landed those changes in fx-team independent of this. I also updated AppConstants to include/use the new code.

**Comment by wesj** [comment]
Testing grunt export this patch seems to work fine on central. I think we end up with some duplicated color values, but I'm willing to live with that for now.

File: app/src/main/res/values/search_dimens.xml [hunk 1]
```diff
@@ -23,4 +23,18 @@
...
+    <dimen name="widget_button_padding">45dp</dimen>
+    <dimen name="widget_text_size">15sp</dimen>
+    <dimen name="widget_drawable_padding">2dp</dimen>
+
+    <!-- Widget Buttons -->
+    <dimen name="widget_header_height">70dp</dimen>
+    <dimen name="widget_button_offset">-50dp</dimen>
+    <dimen name="widget_button_padding">45dp</dimen>
+    <dimen name="widget_text_size">15sp</dimen>
+    <dimen name="widget_drawable_padding">2dp</dimen>
```
Comment (leibovic) [inline app/src/main/res/values/search_dimens.xml:38]: You accidentally declared all these values twice. You should just declare them once :)

**Comment by leibovic** [comment]
Closing this, since this patch was merged in a different PR.

**Pull request closed without merging**